lib/index.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
         // shared
         ConfigOps,
         ConfigValidator,
-        ModuleResolver,
+        resolve,
```

Comment:
to avoid breaking something, please keep the original exported name.

Reply:
Although this change looks like it is breaking, consider this:

Before the PR:

```js
const {ModuleResolver} = require("eslint")

ModuleResolver
//=> undefined
```

After the PR:

```js
const {ModuleResolver} = require("eslint")

ModuleResolver
//=> undefined
```

Since it isn't breaking, we can export it under any name.